### ISSUE TITLE
Clarify licensing of API source code

### ISSUE THREAD

**Issue by argv-minus-one** [JavaMoney/jsr354-api#67]
The licenses for the API do not seem to grant permission to distribute the API jar with an application, other than for “internal evaluation”. Without that permission, there is no way to distribute an application that uses JavaMoney without infringing on Credit Suisse's intellectual property.

**Comment by argv-minus-one** [comment]
The licenses also don't seem to grant permission to GitHub to publish this repo, nor for others to fork this repo. As a result, this repo may be in violation of the [GitHub terms of service, section D.3](https://help.github.com/articles/github-terms-of-service/#3-ownership-of-content-right-to-post-and-license-grants).

**Comment by keilw** [comment]
@atsticks is about to address this with the current Spec Lead company (Credit Suisse). Please also see how JSR 107 was struggeling with a similar issue: https://github.com/jsr107/jsr107spec The license.txt and license headers were changed at least a year ago, but the MR ballot pased only a few days ago: https://jcp.org/en/jsr/results?id=6087

**Comment by argv-minus-one** [comment]
Apologies as I don't understand the JCP very much, but what does it have to do with the license published on GitHub? Was the Apache 2.0 license for JCache somehow invalid until the MR passed?

**Comment by keilw** [comment]
Yes, that's exactly the case, until MR1 the API was licensed under the Oracle/Sun JCP License that (if interpreted strictly by the word) e.g. would even require you to DESTROY every copy of the software after the evaluation has ended ;-) Spec Leads without a huge legal department on their hand were often pushed in earlier decades to use that license everywhere. Nowadays there is a common understanding this "Shrinkwrap License" more or less applies only to the Spec itself while the APIs and RI/TCK are licensed in almost every case (except Java SE/JDK maybe when it comes to future TCKs) under recognized Open Source licenses like Apache, BSD, MIT or Eclipse.

**Comment by keilw** [comment]
Based on their contributions to the API and/or Maintenance Lead roles, could @atsticks, @otaviojava, @chrisphe, @marschall, @reallyinsane, @dunschtig and @BenMorel please signal (with thumbs-up) if you are happy with the API using the Apache License. @McPringle only made one contribution, but being vocal at conferences and also a JCP member, of course your voice is equally welcome.

There should be no -1 votes, otherwise we have to discuss it in detail if someone objected to this change. The Maintenance Leads have a binding vote, officially there is no EG at this point, but everyone who made a significant enough contribution to this repository should have a say if they care.

Thanks,
Werner
